schema/artwork/index.js

```diff
@@ -44,13 +48,28 @@ const ArtworkType = new GraphQLObjectType({
       date: {
         type: GraphQLString,
       },
-      can_share_image: {
-        type: GraphQLBoolean,
-      },
       partner: {
         type: Partner.type,
         resolve: ({ partner }) => gravity(`partner/${partner.id}`),
       },
+      can_share_image: {
+        type: GraphQLBoolean,
+        deprecationReason: 'Favor `is_`-prefixed boolean attributes',
```

Comment:
Looks like the show2 page. By deprecating it still works but no longer shows up in GraphiQL